packages/arb-token-bridge-ui/tests/e2e/specs/depositCctp.cy.ts

```diff
@@ -12,7 +12,7 @@ import {
 import { CommonAddress } from '../../../src/util/CommonAddressUtils'
 
 describe('Deposit USDC through CCTP', () => {
-  const USDCAmountToSend = 0.0001
+  const USDCAmountToSend = Number((Math.random() * 0.001).toFixed(6)) // randomize the amount to be sure that previous transactions are not checked in e2e
```

Comment:
Christophe has enhanced the CCTP e2e in his PR. 
We might need to revert these changes depending on which goes first.